app/controllers/VatDetailsController.scala

```diff
@@ -204,16 +199,16 @@ class VatDetailsController @Inject()(val enrolmentsAuthService: EnrolmentsAuthSe
 
 
   private[controllers] def retrieveEmailVerifiedIfExist(accountDetails: HttpGetResult[CustomerInformation]): Boolean = {
-    accountDetails.fold(_ => true, customerInfo =>
-      customerInfo.emailAddress match {
-        case Some(emailInformation) =>
-          (emailInformation.email, emailInformation.emailVerified) match {
-            case (Some(_), Some(verified)) => verified
-            case (Some(_), None) => false
-            case _ => true
-          }
-        case _ => true
-      })
+    accountDetails match {
+      case Right(details) =>
+        details.emailAddress match {
+          case _ if details.hasPendingPpobChanges => true
```

Comment:
This will prevent users with any ppob inflight changes from seeing the email nudge - is that what we want or do we want to do this only for inflight email changes?